lib/src/interactions.dart

```diff
@@ -6,9 +6,9 @@ import 'package:flutter_test/flutter_test.dart';
 /// An interaction to perform while rendering a golden test. Returns
 /// an asynchronous callback that should be called to cleanup when
 /// the golden test completes.
-typedef Interaction = Future<AsyncCallback?> Function(WidgetTester);
+typedef Interaction = Future<AsyncCallback> Function(WidgetTester);
```

Comment:
not sure if there was any case in which we wanted a nullable callback, so I made it non-nullable as the comment says that it **should** be called when the test completes

Reply:
doing this would make this a breaking api change technically